Extensions/AboutUsParser.js

```diff
@@ -0,0 +1,325 @@
+//regex link for paragraph searching: https://regex101.com/r/UhgGAR/1
+const fs = require('fs');
+const Path = require('path');
+const glob = require('glob');
+const ScriptDir = __dirname; // The directory of where this current javascript is located
+const readlineSync = require('readline-sync');
+
+
+const C = {
+    ModulePath: ScriptDir + '/../source/files/AboutUs/', // Directory path of AboutUs files
+    ComponentsPath: ScriptDir + '/../components/', // Components folder path
+    separator: '[Section]', // Separator used in text files to separate each paragraph
+    jsonFile: 'AboutUs.json',
+    componentName: 'about',
+};
+
+var getFolders = source => fs.readdirSync(Path.join(source))
+    .filter((file) => fs.statSync(Path.join(source, file)).isDirectory());
+
+var getFiles = ((source, callback) => {
+    glob(source + '/**/*', callback);
+});
+
+
+class Paragraph {
+    /**
+     * 
+     * @param {Object} object 
+     */
+    constructor(object) {
+        this.title = object.title;
+        this.short = object.text ? object.text.short : object.short;
+        this.long = object.text ? object.text.long : object.long;
+        this.language = object.language;
+        this.folder = object.folder;
+        this.photo = object.photo ? object.photo : null;
+    }
+
+    getDefaultHtmlCode() {
```

Comment:
Because as things might get added, the html code we would like the parser to produce may change from thing to thing... so that everyone can add html output of their own to fit their needs, but this one I have added is the default one to be used if you would like to produce html just like that one on AboutUs page.